src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -168,20 +169,14 @@ const DateTimeBlock : React.FC<IProps> = ({
     }
   },[date, allowAfterMidnight]);
 
-  const onBlurInputs = useCallback(() =>{
-    if(displayHours === '0' || displayHours === ''){
-      setDisplayHours('00');
+  const onBlurInputs = useCallback((blurValue: string, value:string) =>{
```

Comment:
`value` is being used here to to tell use if we're usings minutes or hours. However the variable name implies that it is a value (like a minute / hour) rather than what it is. Could you give these both names that best match their intention?